crates/bevy_ecs/src/lib.rs

```diff
@@ -1749,16 +1749,4 @@ mod tests {
     #[allow(dead_code)]
     #[derive(Component)]
     struct ComponentB(u32);
-
-    #[derive(Bundle)]
```

Comment:
These should be marked as allow dead code instead like their siblings IMO. These are compilation tests primarily.

Reply:
Sounds good! Added back :)